.github/workflows/security-scan.yml

```
@@ -0,0 +1,87 @@
+      DOCKER_APPLICATION_TAG: latest
+    outputs:
+      alerts_info: ${{ steps.report.outputs.alerts_info }}
+      alerts_low: ${{ steps.report.outputs.alerts_low }}
+      alerts_medium: ${{ steps.report.outputs.alerts_medium }}
+      alerts_high: ${{ steps.report.outputs.alerts_high }}
+      alerts_total: ${{ steps.report.outputs.alerts_total }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup dependencies
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:4:13: Double quote to prevent globbing and word splitting

(shellcheck)